### ISSUE TITLE
[BUG] Prusa XL LCD error indicating heaters turned off due to no activity for 30 minutes, print head moving anyway as if printing

### ISSUE THREAD

**Issue by hillsandales** [prusa3d/Prusa-Firmware-Buddy#3727]
Please, before you create a new bug report, please make sure you searched in open and closed issues and couldn't find anything that matches.

**Printer type** - [XL]

**Printer firmware version** - [5.1.2]

**Original or Custom firmware** - [Original]

**Optional upgrades** - [none]

**USB drive or USB/Octoprint**
  USB drive, reputable brand

**Describe the bug**
  Checked on my print at about 1hr 30 min in, and found it was not extruding filament. LCD screen showed an error stating that the heaters were turned off due to no activity for 30 minutes. However, the printer was still moving the tool head as if it were printing. Indeed, heaters had turned off. Tool and bed were cool. Cancelled the print… Primt was slices to print a few parts at a time but sequentially using option in PrusaSlicer advanced settings. First object was fine. Failed on second object. No errors received. Printer kept moving…looked fine on webcam.

**How to reproduce**
I’m not sure. Haven’t had this happen.

**Expected behavior**
  If heaters turn off, I’d expect print to go to paused state. Perhaps issue an alert via phone or audible tone. With heaters off, machine should not continue to move as if it’s printing fine.

![IMG_0069](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/864901/210c2c47-1f1c-4ca6-8643-05cc1fa6401a)

**Comment by hillsandales** [comment]
Tried running the same file again and skipping the part it finished. Skipping the first part eventually worked. (Stopped part, but Z increased ~5mm and the machine looked like it was printing that part in mid air. It eventually moved to the second part, but when  it started printing, it didn't extrude for the first perimeter.)

Regardless, the 2nd part failed with the same error as before. It looked like it failed at the same layer but I didn't take note of the layer each time. Tried running dump to file after cancelling the print, but it said no dump file was available.

Image of the error on the LCD screen. Printer kept moving as if printing following this error.
![PXL_20240205_050353149](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/864901/a9a08e97-9366-4a9c-812f-bd42720daf24)

**Comment by hillsandales** [comment]
Ran the file a 3rd time without the first part sliced. Parts printed okay, but the machine reported a problem with the tool head docking itself. I think it was switching to Tool 2 to print the remaining parts in a different color. Tool 1 got stuck inside the model that had printed. Again, I had sliced the file with the Output options > Complete individual objects checked. It seems there might be a bug somewhere in the slicer or the firmware for this object. The XL doesn't know how to handle it, and it caused a crash when switching to tool 2 to print the last part. The extruder clearance was left at the default for the XL profile (45mm radius, 14mm height).
